src/FixerDefinition/CodeSampleInterface.php

```diff
@@ -26,4 +26,9 @@ public function getCode();
      * @return null|array
      */
     public function getConfiguration();
+
+    /**
+     * @return null|int
+     */
+    public function getRequiredVersion();
```

Comment:
Hmm, depends how much we want to respect the BC promise. Maybe it's worth making a 3.x branch for this? I don't think we should just add more interfaces, though, that's maybe not such a bad idea - would definitely be a solution.

Really something for @keradus for decide. Each solution has pros and cons, and the release timeline for 3.x affects things.